src/components/EntityPage/File/index.js

```diff
@@ -330,12 +327,7 @@ const FileEntity = ({ api, fileId }) => {
               <EntityContent>
                 <EntityContentSection title="File Properties">
                   <Row style={{ width: '100%' }}>
```

Comment:
completely just a nit pick :P can we do `className={css'...'}` to stick with emotion? :)